pkg/oam/util/helper.go

```diff
@@ -620,6 +621,19 @@ func RawExtension2Unstructured(raw *runtime.RawExtension) (*unstructured.Unstruc
 	}, nil
 }
 
+// RawExtension2CompManifest converts a rawExtension to ComponentManifest
+func RawExtension2CompManifest(raw *runtime.RawExtension) (*oamtypes.ComponentManifest, error) {
```

Comment:
Does it really be used?  Please write test for it

Reply:
No longer used since we remove it out of CRD. I will delete it.